snuba/clickhouse/http.py

```diff
@@ -39,12 +39,38 @@ def __default(self, value: Any) -> Any:
         else:
             raise TypeError
 
-    def encode(self, value: WriterTableRow) -> JSONRow:
+    def encode(self, value: WriterTableRow) -> bytes:
```

Comment:
These types are equivalent, but I didn't like seeing JSONRow referring to byte strings that are not JSON